experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -3593,6 +3593,65 @@ def test_firefox_labs_required_fields(
             }
             self.assertEqual(expected_errors, serializer.errors)
 
+    @parameterized.expand(
+        chain(
+            (
+                (application, group, required_version, None)
+                for application, available_groups in (
+                    NimbusExperiment.FIREFOX_LABS_GROUP_AVAILABILITY.items()
+                )
+                for group, required_version in available_groups.items()
+            ),
+            (
+                (
+                    application,
+                    group,
+                    f"{parsed_version.major - 1}.0.0",
+                    NimbusExperiment.ERROR_FIREFOX_LABS_GROUP_MIN_VERSION.format(
+                        version=parsed_version
+                    ),
+                )
+                for application, available_groups in (
+                    NimbusExperiment.FIREFOX_LABS_GROUP_AVAILABILITY.items()
+                )
+                for group, required_version in available_groups.items()
+                for parsed_version in (NimbusExperiment.Version.parse(required_version),)
+                if required_version != NimbusExperiment.Version.FIREFOX_137
+            ),
+        )
+    )
+    def test_firefox_labs_group_availability(
+        self, application, firefox_labs_group, version, expected_error
+    ):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            application=NimbusExperiment.Application.DESKTOP,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_137,
```

Comment:
Am I misreading this?  Why do we do maj_version -1 as a parameter and then hardcode 137 here?